src/seedu/addressbook/data/tag/UniqueTagList.java

```diff
@@ -94,18 +94,6 @@ public boolean contains(Tag toCheck) {
     }
 
     /**
-     * Adds a Tag to the list.
-     *
-     * @throws DuplicateTagException if the Tag to add is a duplicate of an existing Tag in the list.
-     */
-    public void add(Tag toAdd) throws DuplicateTagException {
```

Comment:
Actually I meant having a separate commit in this current PR, not a separate PR.